components/omega/src/infra/TimeMgr.cpp

```diff
@@ -3947,7 +3985,7 @@ std::string TimeInstant::getString(
    FmtString << ":";
    FmtString << "%02d"; // 2 digit minute with leading 0
    FmtString << ":";
-   FmtString << "%" << SecondWidth + 3 << "." << SecondWidth << "f"; // seconds
+   FmtString << "%0" << SecondWidth + 3 << "." << SecondWidth << "f"; // seconds
```

Comment:
Based on @hyungyukang experience, we may want to modify this so that when SecondWidth is zero, we change the format to a zero-prepadded 2-digit integer like in minutes, hours.

Reply:
Thanks, I wasn't sure what the precise issue was. I opted for "%02.0d", since the Seconds value is an R8